src/Cli/OptionCommand.php

```diff
@@ -0,0 +1,383 @@
+<?php
+/**
+ * Class OptionCommand.
+ *
+ * Commands that deal with the AMP options.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP\Cli;
+
+use WP_CLI;
+use WP_Error;
+use AmpProject\AmpWP\Option;
+use AmpProject\AmpWP\Admin\ReaderThemes;
+use AmpProject\AmpWP\Infrastructure\Service;
+use AmpProject\AmpWP\Infrastructure\CliCommand;
+
+/**
+ * Retrieves and sets AMP plugin options.
+ *
+ * ## EXAMPLES
+ *
+ * # Get AMP plugin option.
+ * $ wp amp option get theme_support
+ * standard
+ *
+ * # Update AMP plugin option.
+ * $ wp amp option update theme_support reader
+ * Success: Updated theme_support option.
+ *
+ * # List AMP plugin options.
+ * $ wp amp option list
+ * +------------------+----------------+
+ * | key              | value          |
+ * +------------------+----------------+
+ * | theme_support    | standard       |
+ * | mobile_redirect  | disabled       |
+ * | reader_theme     | legacy         |
+ * +------------------+----------------+
+ *
+ * @since 2.4.0
+ */
+final class OptionCommand implements Service, CliCommand {
+
+	/**
+	 * Options endpoint.
+	 *
+	 * @var string
+	 */
+	const OPTIONS_ENDPOINT = '/amp/v1/options';
+
+	/**
+	 * Allowed options to be managed via the CLI.
+	 *
+	 * @var string[]
+	 */
+	const ALLOWED_OPTIONS = [
+		Option::READER_THEME,
+		Option::THEME_SUPPORT,
+		Option::MOBILE_REDIRECT,
```

Comment:
I just realized that there is one other setting which would be quite helpful and that is whether sandboxing is enabled and the selected sandboxing level. Could you add those as well?

Reply:
Oh, one more: Delete plugin data at uninstall